person.rb

```diff
@@ -19,6 +21,11 @@ def correct_name
     @name
   end
 
+  def add_rentals=(rental)
+    rental.person = self
+    rentals.push(rental)
+  end
```

Comment:
- [ ] The same thing with the `add_rentals` method in the Person class. The person itself should be sent as a parameter to create the new rental achieving the 'has-many' association.

```ruby
def add_rental(book, date)
    Rental.new(date, book, self)
end
```